src/test/java/wheresmymoney/ExpenseListTest.java

```diff
@@ -105,4 +107,32 @@ public void deleteExpense_emptyList_throwsWheresMyMoneyException() {
                 () -> expenseList.deleteExpense(1));
     }
 
+    @Test
+    public void listByCategory_normalListByCategory_success() {
+        ExpenseList expenseList = new ExpenseList();
+        Expense[] sampleArray = {};
+        Expense expense = new Expense(1.00f, "Ice Cream", "Food");
+        expenseList.addExpense(1.00f, "Ice Cream", "Food");
+        expenseList.addExpense(4.50f, "Taxi", "Transport");
+        assertEquals(1, expenseList.listByCategory("Food").size());
+        assertEquals(expense.getDescription(),
+                expenseList.listByCategory("Food").toArray(sampleArray)[0].getDescription());
```

Comment:
You can just use the get method of the ArrayList instead of converting the ArrayList into an Array